src/components/TopBar/tests/TopBar.test.tsx

```diff
@@ -170,62 +171,67 @@ describe('<TopBar />', () => {
     });
   });
 
-  describe('logo', () => {
-    it('will render an image with the logo top bar source', () => {
-      const topBar = mountWithApp(<TopBar />, {
-        theme: {
-          logo: {
-            topBarSource: './assets/shopify.svg',
-          },
-        },
-      });
-      expect(topBar).toContainReactComponent(Image, {
-        source: './assets/shopify.svg',
-      });
-    });
-
-    it('will render an image with the logo accessibility label', () => {
-      const topBar = mountWithApp(<TopBar />, {
-        theme: {
-          logo: {
-            accessibilityLabel: 'Shopify',
-          },
-        },
-      });
-      expect(topBar).toContainReactComponent(Image, {
-        alt: 'Shopify',
-      });
-    });
-
-    it('will render an unstyled link with the logo URL', () => {
-      const topBar = mountWithApp(<TopBar />, {
-        theme: {logo: {url: 'https://shopify.com'}},
-      });
-
-      expect(topBar).toContainReactComponent(UnstyledLink, {
-        url: 'https://shopify.com',
-      });
-    });
-
-    it('will render an unstyled link with the logo width', () => {
-      const topBar = mountWithApp(<TopBar />, {
-        theme: {logo: {width: 124}},
-      });
-
-      expect(topBar).toContainReactComponent(UnstyledLink, {
-        style: {width: '124px'},
-      });
-    });
-
-    it('will render an unstyled link with a default width', () => {
-      const topBar = mountWithApp(<TopBar />, {
-        theme: {logo: {}},
-      });
-      expect(topBar).toContainReactComponent(UnstyledLink, {
-        style: {width: '104px'},
-      });
-    });
-  });
+  // describe('logo', () => {
```

Comment:
Drawing attention here to ensure this is capture in the following PR: https://github.com/Shopify/polaris-react/pull/4667

Reply:
Silly nit:

```js
  describe.skip('logo', () => {
```

Would let you skip the tests without having to comment all of them out

Reply:
For sure 👍  This we be immediately re-enabled in Kyles upcoming PR